client/landing/gutenboarding/onboarding-block/acquire-intent/index.tsx

```diff
@@ -1,48 +1,31 @@
-import config from '@automattic/calypso-config';
 import { isGoodDefaultDomainQuery } from '@automattic/domain-picker';
 import { ActionButtons, BackButton, NextButton, SkipButton } from '@automattic/onboarding';
-import { useViewportMatch } from '@wordpress/compose';
 import { useSelect, useDispatch } from '@wordpress/data';
 import { useI18n } from '@wordpress/react-i18n';
 import * as React from 'react';
 import useDetectMatchingAnchorSite from '../../hooks/use-detect-matching-anchor-site';
 import useFseBetaOptInStep from '../../hooks/use-fse-beta-opt-in-step';
 import useStepNavigation from '../../hooks/use-step-navigation';
 import { useTrackStep } from '../../hooks/use-track-step';
-import { recordVerticalSkip, recordSiteTitleSkip } from '../../lib/analytics';
+import { recordSiteTitleSkip } from '../../lib/analytics';
 import { useIsAnchorFm } from '../../path';
 import { STORE_KEY } from '../../stores/onboard';
-import Arrow from './arrow';
 import SiteTitle from './site-title';
-import VerticalSelect from './vertical-select';
 
 import './style.scss';
 
 const AcquireIntent: React.FunctionComponent = () => {
 	const { __ } = useI18n();
-	const {
-		getSelectedVertical,
-		getSelectedSiteTitle,
-		wasVerticalSkipped,
-		hasSiteTitle,
-	} = useSelect( ( select ) => select( STORE_KEY ) );
+	const { getSelectedVertical, getSelectedSiteTitle, hasSiteTitle } = useSelect( ( select ) =>
+		select( STORE_KEY )
+	);
```

Comment:
Is this still needed?
```suggestion
	const { getSelectedSiteTitle, hasSiteTitle } = useSelect( ( select ) =>
		select( STORE_KEY )
	);
```

Reply:
Nope, good find 👍